knit/core.py

```diff
@@ -101,6 +101,7 @@ def __init__(self, autodetect=True, upload_always=False, hdfs_home=None,
         self.upload_always = upload_always
         self.hdfs_home = hdfs_home or self.conf.get('dfs.user.home.base.dir',
                                                '/user/' + self.conf.get('user', conf['user']))
+        self.lang = self.conf.get('lang', 'C.UTF-8')
```

Comment:
So the intent is that `lang` can be set as with any Knit/dask-yarn parameter, but it isn't explicitly in any parameter list, and in general missing?
We may need to start a "Troubleshooting" section for the docs to note the various things than can be checked or changed.

Reply:
Yeah, I think this has a bad smell, but I am not sure where it should go. Maybe an explicit parameter to both `DaskYarnCluster` and `Knit` that is documented in both `__init__` docstrings?